#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Change some locales LC_MONETARY symbols.
# DP:		ar_SD (SDP -> SDD), sr_YU (YUN -> YUM)
# DP:		ar_SD: #173963	Patched by era eriksson <era@iki.fi>.
# DP:		sr_YU* and es_EC: #160040 Patched by gotom@debian.org.
# DP: Author: GOTO Masanori <gotom@debian.org>
# DP: Upstream status: Pending
# DP: Status Details: These patches were submitted but not accepted because
# DP:		their symbols have been well established yet.
# DP: Date: 2003-04-01

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- localedata/locales/ar_SD	Thu Oct 26 20:44:45 2000
+++ localedata/locales/ar_SD	Thu Mar 20 12:41:32 2003
@@ -44,10 +44,10 @@
 
 LC_MONETARY
 % This is the POSIX Locale definition the LC_MONETARY category.
-% These are generated based on XML base Locale difintion file
+% These are generated based on XML base Locale defintion file
 % for IBM Class for Unicode/Java
 %
-int_curr_symbol       "<U0053><U0044><U0050><U0020>"
+int_curr_symbol       "<U0053><U0044><U0044><U0020>"
 currency_symbol       "<U062C><U002E><U0633><U002E>"
 mon_decimal_point     "<U002E>"
 mon_thousands_sep     "<U002C>"
--- localedata/locales/sr_YU	2002-09-09 13:48:52.000000000 +0900
+++ localedata/locales/sr_YU.new	2002-09-09 13:50:54.000000000 +0900
@@ -224,7 +224,7 @@
 END LC_MESSAGES
 
 LC_MONETARY
-int_curr_symbol           "<U0059><U0055><U004E><U0020>"
+int_curr_symbol           "<U0059><U0055><U004D><U0020>"
 currency_symbol           "<U0064><U0069><U006E>"
 mon_decimal_point         "<U002C>"
 mon_thousands_sep         "<U0020>"
--- localedata/locales/sr_YU@cyrillic	2002-09-09 13:48:52.000000000 +0900
+++ localedata/locales/sr_YU@cyrillic.new	2002-09-09 13:50:43.000000000 +0900
@@ -61,7 +61,7 @@
 END LC_MESSAGES
 
 LC_MONETARY
-int_curr_symbol           "<U0059><U0055><U004E><U0020>"
+int_curr_symbol           "<U0059><U0055><U004D><U0020>"
 currency_symbol           "<U0434><U0438><U043D>"
 mon_decimal_point         "<U002C>"
 mon_thousands_sep         "<U0020>"
